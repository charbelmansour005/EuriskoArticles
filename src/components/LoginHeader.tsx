import {StyleSheet, Text, Image, Dimensions} from 'react-native';
import React from 'react';
import Logo from '../../assets/eurisko.jpg';

const LoginHeader = (): JSX.Element => {
  return (
    <>
      <Image style={styles.LogoCentered} source={Logo} />
      <Text style={styles.WelcomeMessage}>Welcome back!</Text>
    </>
  );
};

export default LoginHeader;

const width = Dimensions.get('window').width;
const height = Dimensions.get('window').height;

const styles = StyleSheet.create({
  LogoCentered: {
    width: width > 350 ? 80 : 60,
    height: height > 350 ? 94 : 60,
    borderRadius: 5,
    borderWidth: 4,
    borderColor: 'black',
  },
  WelcomeMessage: {
    fontWeight: 'bold',
    fontSize: 20,
    color: 'black',
    marginVertical: 8,
  },
});
