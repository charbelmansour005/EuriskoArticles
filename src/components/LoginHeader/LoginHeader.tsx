import {StyleSheet, Image, Dimensions} from 'react-native';
import React from 'react';
import Logo from '../../../assets/eurisko.jpg';

const LoginHeader = (): JSX.Element => {
  return <Image style={styles.LogoCentered} source={Logo} />;
};

export default LoginHeader;

const width = Dimensions.get('window').width;
const height = Dimensions.get('window').height;

const styles = StyleSheet.create({
  LogoCentered: {
    width: width > 350 ? 90 : 70,
    height: height > 350 ? 104 : 70,
    borderRadius: 5,
    borderWidth: 0,
    borderColor: 'black',
    marginBottom: '10%',
  },
});
