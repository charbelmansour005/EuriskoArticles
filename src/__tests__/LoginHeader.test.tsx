import React from 'react'
import renderer from 'react-test-renderer'
import LoginHeader from '../components/LoginHeader/LoginHeader'

test('renders correctly', async () => {
  const tree = renderer.create(<LoginHeader />).toJSON()
  expect(tree).toMatchSnapshot()
})
