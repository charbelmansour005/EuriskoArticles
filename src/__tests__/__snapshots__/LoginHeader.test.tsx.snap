// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  source={
    Object {
      "testUri": "../../../assets/eurisko.jpg",
    }
  }
  style={
    Object {
      "borderColor": "black",
      "borderRadius": 5,
      "borderWidth": 0,
      "height": 104,
      "marginBottom": "10%",
      "width": 90,
    }
  }
  testID="image"
/>
`;
